packages/website-frontend/angular.json

```diff
@@ -41,7 +41,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "500kb",
-                  "maximumError": "1mb"
+                  "maximumError": "1.6mb"
```

Comment:
😄

Reply:
Yeah... It's a bit much. Ideally there'd be very little JS on pages that don't need it? Maybe if you get rid of Angular you could look into https://astro.build/ with dynamic content rendered in lit?